### PR TITLE
docs round 1

### PR DESCRIPTION
**Pull request by chadfurman**
First round of docs, `jekyll new` scaffolding with some light markdown

To test, make sure you have jekyll installed, clone down, and run `jekyll build && jekyll serve --watch`

### CODE REVIEW

File: docs/advanced-queries.md [hunk 1]
```diff
@@ -0,0 +1,1318 @@
...
+
+* Move table sql utility function
+
+* Clean up catalog API
+
+* Give get table sql function its own file
+
+* Add another exapmle function
+
+* Make internal
```
Comment (benjie) [inline docs/advanced-queries.md]: I was expecting markdown...?

**Comment by benjie** [comment]
@chadfurman From scanning the diff I don't think the existing docs have come through correctly?

**Comment by chadfurman** [comment]
Try to build it locally. Check the links on the page. It should all be formatted correctly

**Comment by chadfurman** [comment]
@benjie

**Comment by chadfurman** [comment]
@benjie you were right, actually.  My bad.  I've fixed it.

**Review by benjie — APPROVED** [review]

**Comment by benjie** [comment]
Looked good to me scanning the diff; have merged and we can fix issues from there.

**Pull request merged**